src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs

```diff
@@ -94,6 +94,8 @@ private void InitializeNavigationViewItems()
 				AddMenuItem<CardSamplePage>(icon: "CardsIcon");
 				AddMenuItem<CheckBoxSamplePage>(icon: "CheckboxIcon");
 				AddMenuItem<ComboBoxSamplePage>(icon: "ComboBoxIcon");
+				// TODO Add divider icon?
```

Comment:
I'm not sure what to do with this?  Do we have an icon that I can use here?

Reply:
I think you should ask design for a new one?
@MatFillion  maybe we can use to have a "default" icon to use for new controls until we can get to an update of the list ?

Reply:
I asked design to give me a default assets. I'm going to add it on another PR